### PR TITLE
container_create: Don't save additional copies of config.json

### PR DESCRIPTION
**Pull request by mrunalp**
Signed-off-by: Mrunal Patel <mpatel@redhat.com>

### CODE REVIEW

File: server/container_create.go [hunk 1]
```diff
@@ -766,9 +766,6 @@ func (s *Server) createSandboxContainer(ctx context.Context, containerID string,
 	specgen.SetRootPath(mountPoint)
 
 	saveOptions := generate.ExportOptions{}
-	if err = specgen.SaveToFile(filepath.Join(containerInfo.Dir, "config.json"), saveOptions); err != nil {
```
Comment (runcom) [inline server/container_create.go]: don't we need this in `Dir` when rebooting a machine? `RunDir` will go away? (can't fully check what I'm saying /cc @nalind ). Point I think is if we don't save config in /var/lib, we can't restore containers

Comment (runcom) [inline server/container_create.go]: well, nevermind, integration tests should catch this - let's see

**Comment by runcom** [comment]
Just a minor point, if Travis is happy, I am as well :)

**Comment by mrunalp** [comment]
This needs more work. I'll open a new one. Thanks!

File: server/container_create.go (hunk 1 above)
Comment (runcom) [inline server/container_create.go]: @mrunalp I think the correct PR would drop `RunDir` config.json. We'll always read from `Dir` instead which works with restore after reboot. We need to not read from RunDir anymore though, @nalind is this something feasible? Does container create work regardless?

**Pull request closed without merging**